pkg/services/featuremgmt/registry.go

```diff
@@ -284,7 +284,8 @@ var (
 		{
 			Name:        "topnav",
 			Description: "New top nav and page layouts",
-			State:       FeatureStateAlpha,
+			State:       FeatureStateBeta,
+			Expression:  "true", // enabled by default
```

Comment:
@ryantxu I copied `Expression:  "true", // enabled by default` from another feature in this file but it does not appear to work (it's not enabled by default )

Reply:
Check the failing backend test:  
<img width="825" alt="image" src="https://user-images.githubusercontent.com/705951/211852047-bdec6e9f-4b07-4133-9256-43bd5530c7e9.png">

Currently, only flags marked "stable" can be enabled by default -- we can obviously change that.